x-pack/plugins/security_solution/public/management/components/console/components/console_state/state_update_handlers/handle_execute_command.tsx

```diff
@@ -133,6 +148,19 @@ export const handleExecuteCommand: ConsoleStoreReducer<
     commandDefinition,
   };
   const requiredArgs = getRequiredArguments(commandDefinition.args);
+  const exclusiveOrArgs = getExclusiveOrArgs(commandDefinition.args);
+
+  const exclusiveOrErrorMessage = (
+    <FormattedMessage
+      id="xpack.securitySolution.console.commandValidation.exlcusiveOr"
+      defaultMessage="This command supports one and only one of the following arguments: {argNames}"
```

Comment:
`...supports only one of the following arguments...` should be clear enough. No?

Reply:
the above text change suggestions added here: https://github.com/elastic/kibana/pull/135260/commits/62b54f17ad4bc0ece345733c0c7c131c28511828